content/concepts/the-rippled-server/online-deletion.md

```diff
@@ -0,0 +1,126 @@
+
+
+## Configuration
+
+The following settings relate to online deletion:
+
+- **`online_delete`** - Specify a number of validated ledger versions to keep. The server periodically deletes any ledger versions that are older than this number. If not specified, no ledgers are deleted.
+
+    The default config file specifies 2000 for this value. This cannot be less than 256, because some events like [Fee Voting](fee-voting.html) and the [Amendment Process](amendments.html#amendment-process) update only every 256 ledgers.
+
+    **Caution:** If you run `rippled` with `online_delete` disabled, then later enable `online_delete` and restart the server, the server disregards but does not delete existing ledger history that your server already downloaded while `online_delete` was disabled. To save disk space, delete your existing history before re-starting the server with `online_delete` back on.
+
+- **`[ledger_history]`** - Specify a number of validated ledgers, equal to or less than `online_delete`. If the server does not have at least this many validated ledger versions, it attempts to backfill them by fetching the data from peers.
+
+    The default for this setting is 256 ledgers.
+
+    The following diagram shows the relationship between `online_delete` and `ledger_history` settings:
+
+    ![Ledgers older than `online_delete` are automatically deleted. Ledgers newer than `ledger_history` are backfilled. Ledgers in between are kept if available but not backfilled](img/online_delete-vs-ledger_history.png)
+
+- **`advisory_delete`** - If enabled, online deletion is not scheduled automatically. Instead, an administrator must manually trigger online deletion. Use the value `0` for disabled or `1` for enabled.
+
+    This setting is disabled by default.
+
+- **`[fetch_depth]`** - Specify a number of ledger versions. The server does not accept fetch requests from peers for historical data that is older than the specified number of ledger versions. Specify the value `full` to serve any available data to peers.
+
+    The default for `fetch_depth` is `full` (serve all available data).
+
+    The `fetch_depth` setting cannot be higher than `online_delete` if both are specified. If `fetch_depth` is set higher, the server treats it as equal to `online_delete` instead.
+
+    The following diagram shows how fetch_depth works:
+
+    ![Ledger versions older than fetch_depth are not served to peers](img/fetch_depth.png)
+
+For estimates of how much disk space is required to store different amounts of history, see [Capacity Planning](capacity-planning.html#historical-data).
+
+### Advisory Deletion
+
+By default, online deletion happens automatically and periodically. If the `advisory_delete` setting is enabled, online deletion only happens when an administrator triggers it using the [can_delete method][]. You can run this command with a scheduled job to trigger automatic deletion based on clock time instead of the number of ledger versions closed.
```

Comment:
If online_delete is configured, it happens automatically. If not configured (the default), then it doesn't. advisory_delete has no meaning if it's not configured.

And can_delete can be run manually to maintain as much history is desired, and to delete only those earlier ledgers that you don't want any more.

Reply:
Reworded to make this more precise.